docs/user/troubleshooting/plasma.md

```diff
@@ -0,0 +1,74 @@
+---
+title: Plasma Troubleshooting
+summary: Troubleshooting Issues With the Plasma Desktop
+---
+
+# Plasma Desktop Troubleshooting
+
+This contains fixes for various issues you might encounter with the Plasma desktop environment.
+
+## Desktop no longer responds to clicks, cursor stuck in "move" mode (cross)
+
+This is often caused by a Firefox bug. It can be triggered by various things, like moving Firefox windows.
+
+**Symptoms:**
+- Can't click on anything on the desktop.
+- Bringing up windows with keyboard shortcuts (ex: Super for the app menu, F9 for a drop down terminal) works, but they don't get focused. You can't interact with them.
+- Mouse cursor is stuck in move mode, it looks like a cross.
+
+You may not need to reboot to recover from this state. Killing all firefox processes may be all that's necessary. To do so:
+
+- Get to a virtual terminal (TTY) with Ctrl-Alt-F3
```

Comment:
Use `+` symbols to indicate [keyboard shortcuts](https://developers.google.com/style/ui-elements#keyboard) (Google suggests using the kbd tag, but wrapping everything in inline code blocks should be similar):

```suggestion
1. Open a virtual terminal by pressing `Ctrl+Alt+F3`.
```

Reply:
Done